x-pack/plugins/canvas/public/components/workpad_loader/workpad_loader.js

```diff
@@ -182,17 +184,17 @@ export class WorkpadLoader extends React.PureComponent {
         sortable: true,
         dataType: 'string',
         render: (name, workpad) => {
+          const router = this.context;
           const workpadName = getDisplayName(name, workpad, loadedWorkpad);
 
           return (
-            <Link
+            <EuiLink
+              onClick={() => router.navigateTo('loadWorkpad', { id: workpad.id })}
```

Comment:
can we also add a `target=` here as well so that the command+click will work to open a workpad in a new tab? I think that's a behavior we still want to keep